code/game/gamemodes/cult/runes.dm

```diff
@@ -501,8 +510,8 @@ var/list/sacrificed = list()
 	log_and_message_admins("used a manifest rune.")
 
 	while(this_rune && user && user.stat==CONSCIOUS && user.client && user.loc==this_rune.loc)
-		user.take_organ_damage(1, 0)
-		sleep(30)
+	user.rune_apply_damage(10, 15)

```

Comment:
What is this code. It will break all the things.

This code is meant to read:
```dm
	while(this_rune && user && user.stat==CONSCIOUS && user.client && user.loc==this_rune.loc)
		user.rune_apply_damage(10, 15)
		sleep(30)
```

The call is supposed to block until the condition of the while condition is true. At the moment you replaced this with a basically infinite loop that locks the entire game.